src/userGuide/UserGuideFi.tsx

```diff
@@ -547,26 +543,26 @@ function UserGuideFi(): ReactElement {
       </UserGuideAccordion>
       <h2 id="_Tietojen_poisto">Tietojen poisto</h2>
       <p>
-        Voit poistaa tietosi joko yksittäisistä palveluista tai koko profiilin.
-        Poiston myötä kaikki tietosi palvelusta poistetaan tai anonymisoidaan,
-        jos palvelu esim. lakisääteisenä joutuu niitä säilyttämään. Sinulla ei
-        kuitenkaan ole poiston jälkeen pääsyä tietoihin, eikä niitä ole
-        yhdistettävissä sinuun.
+        Voit poistaa tietosi yksittäisistä palveluista tai halutessasi koko
+        profiilin. Poiston myötä kaikki tietosi palvelusta poistetaan tai
+        anonymisoidaan, jos palvelu esim. lakisääteisenä joutuu niitä
+        säilyttämään. Sinulla ei kuitenkaan ole poiston jälkeen pääsyä
```

Comment:
En ole äidinkielessä ihan paras, mutta nykyisellään tuo sivulause ("jos palvelu esim. lakisäätiesenä joutuu niitä säilyttämään") viittaa koko edeltävään lauseeseen, eli myös tuohon poistamiseen. En ole ihan varma, miten tämä korjataan; varmaan helpointa olisi jakaa tuo vaan kahteen lauseeseen, esimerkiksi:
```suggestion
        profiilin. Poiston myötä kaikki tietosi palvelusta poistetaan. 
        Jos palvelu joutuu esim. lakisääteisistä syistä säilyttämään 
        tietoja, tiedot anonymisoidaan poiston sijaan. Sinulla ei 
        kuitenkaan ole poiston jälkeen pääsyä
```

Reply:
Jako kahteen lauseeseen paljon selkeämpi.